CMakeLists.txt

```diff
@@ -137,6 +137,8 @@ endif()
 add_compile_options(-Wall)
 add_compile_options(-Wextra)
 add_compile_options(-Wno-unused-parameter)
+add_compile_options(-Wno-ignored-qualifiers)
+add_compile_options(-Wno-return-type)
```

Comment:
Sure. I separated them out anyway to #8, and since i am keeping a separtae branch with all the cherry-picked commits that I am using for our internal testing,  it's no problem for me. It makes perfect sense to keep those (and track those) as separate.